tests/unittests/operators/delegated_tests.py

```
@@ -1130,6 +1130,63 @@ def test_execute_operation_monitor_internal_fail(
         self.assertIn("Operation FAILED (detected by monitor):", result.error)
         self.assertIn("MockOperator failed internally", result.error)
 
+    @patch("logging.handlers.QueueListener")
+    @patch("multiprocessing.get_context")
+    def test_execute_operation_monitor_unclean_exit(
+        self,
+        mock_get_context,
+        mock_listener,
```

Ruff (0.15.12)

[warning] Unused method argument: `mock_listener`

(ARG002)